src/components/AboutVisualizationUnit/AboutVisualizationUnit.js

```diff
@@ -0,0 +1,263 @@
+        },
+    },
+    dataStatistics: {
+        resource: 'dataStatistics/favorites',
+        id: ({ id }) => id,
+    },
+})
+
+const getSubscribeMutation = (type, id) => ({
+    resource: `${type}/${id}/subscriber`,
+    type: 'create',
+})
+
+const getUnsubscribeMutation = (type, id) => ({
+    resource: `${type}/${id}/subscriber`,
+    type: 'delete',
+})
+
+const AboutVisualizationUnit = ({ type, id }) => {
+    const [isExpanded, setIsExpanded] = useState(true)
+
+    const queries = useMemo(() => getQueries(type), [])
+
+    const {
+        data,
+        loading: dataIsLoading,
+        refetch,
+    } = useDataQuery(queries, {
+        lazy: true,
+    })
+
+    const subscribeMutation = useMemo(() => getSubscribeMutation(type, id), [])
+
+    const unsubscribeMutation = useMemo(
+        () => getUnsubscribeMutation(type, id),
+        []
+    )
+
+    const [subscribe, { loading: subscribeIsLoading }] = useDataMutation(
+        subscribeMutation,
+        {
+            onComplete: res => {
+                if (res.status === 'OK') {
+                    refetch({ id })
+                }
+            },
+        }
+    )
+
+    const [unsubscribe, { loading: unsubscribeIsLoading }] = useDataMutation(
+        unsubscribeMutation,
+        {
+            onComplete: res => {
+                if (res.status === 'OK') {
+                    refetch({ id })
+                }
+            },
+        }
+    )
+
+    useEffect(() => {
+        if (id) {
+            refetch({ id })
+        }
+    }, [type, id])
+
+    const getAccessLevelString = access => {
+        const accessMatch = access.match(/(?<accessLevel>rw?)/)
+
+        switch (accessMatch.groups.accessLevel) {
+            case 'r':
+                return i18n.t('view only')
+            case 'rw':
+                return i18n.t('view and edit')
+        }
+    }
+
+    const getSharingSummary = visualization => {
+        const sharingText = []
+
+        if (/^rw?/.test(visualization.publicAccess)) {
+            sharingText.push(
+                i18n.t('all users ({{accessLevel}})', {
+                    accessLevel: getAccessLevelString(
+                        visualization.publicAccess
+                    ),
+                })
+            )
+        }
+
+        const userAccesses = visualization.userAccesses
+        const groupAccesses = visualization.userGroupAccesses
+
+        userAccesses.concat(groupAccesses).forEach(accessRule => {
+            sharingText.push(
+                i18n.t('{{userOrGroup}} ({{accessLevel}})', {
+                    userOrGroup: accessRule.displayName,
+                    accessLevel: getAccessLevelString(accessRule.access),
+                })
+            )
+        })
+
+        return sharingText.length
+            ? i18n.t('Shared with {{commaSeparatedListOfUsersGroups}}', {
```

Comment:
```suggestion
            ? i18n.t('Shared with {{commaSeparatedListOfUsersAndGroups}}', {
```
its users **and** groups, not usersgroups as a joint entity, correct? 🤔

Reply:
Technically can be either one or both, but yes, since it's seen by the translators `commaSeparatedListOfUsersAndGroups` reads better.
Renamed.